graphistry/plotter.py

```diff
@@ -164,6 +164,7 @@ def bind(self, source=None, destination=None, node=None,
         res._point_label = point_label or self._point_label
         res._point_color = point_color or self._point_color
         res._point_size = point_size or self._point_size
+
```

Comment:
Just as a habit, nice not to have unrelated formatting changes as part of a PR...